### ISSUE TITLE
Unable to save TSVART review assignments

### ISSUE THREAD

**Issue by gloinul** [ietf-tools/datatracker#7842]
### Describe the issue

Was going through unassigned review requests for TSVART. 
https://datatracker.ietf.org/group/tsvart/reviews/manage/unassigned/

After having given one doc to a reviewer and closing the rest I am unable to save. In fact none of the bottom buttons (Save Changes, Save and Continue, and Refresh) appear to be working. I have tried this both in Safari Version 17.6 (19618.3.11.11.5) and Chrome Version 127.0.6533.120 (Official Build) (arm64) getting the same result. I am logged in on both. I have also tried on another computer in Firefox 129.0.1 without success. 
 

 

### Code of Conduct

- [X] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

---
Aggregated with agent assistance on 2026-08-18

**Comment by rjsparks** [comment]
Duplicate of #7839.

@gloinul
Please check something for me. Look for that "Review Type" field and ensure its set to telechat review. If setting that doesn't clear the issue, let me know and we'll get Éric to put in an explicit review request for tsvart.

**Comment by gloinul** [comment]
I don't have the same view or that field indicating a need to select that is shown in #7839. But it might be related as the same document is present in my queue. See screen shot. 
![Screenshot 2024-08-20 at 17 07 25](https://github.com/user-attachments/assets/702c6484-e321-41bd-9197-45b1530987ff)

I do note that that rfc81110-to-ieee do have both telechat and Last call dead line.

**Comment by rjsparks** [comment]
No - I'll get Éric to explicitly request a review as a workaround - you can then say "will not review" without it breaking. Thanks for the screenshot - it gives insight into a part of the workflow that is breaking that wasn't obvious before.

**Comment by gloinul** [comment]
Thanks it worked now, (using refresh button on page), and then I could save with the intended choices for all the documents.

**Comment by rjsparks** [comment]
Keeping this issue open as it highlights a place that the form needs to be corrected.

**Comment by evyncke** [comment]
Do I still need to request a TSVART review ? It seems that Magnus' issue is solved now.

**Comment by gloinul** [comment]
@evyncke no, you don't need to for me. I wasn't going to assign draft-wkumari-rfc81110-to-ieeee to any reviewer.
